config-linux.md

```diff
@@ -147,7 +147,7 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/random`][random.4]
 * [`/dev/urandom`][random.4]
 * [`/dev/tty`][tty.4]
-* [`/dev/console`][console.4]
+* [`/dev/console`][console.4] is setup if terminal is enabled in the config
```

Comment:
437ff74 → d789654 dropped the earlier link between `process.terminal` and `/dev/console`.  Do we want to keep this conditional here?  I'd rather drop the change and require the runtime to always provide `/dev/console` (pointing to whatever console it feels is appropriate).